person.rb

```diff
@@ -0,0 +1,23 @@
+require './person'
+
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
```

Comment:
- Hey, let's also add the missing correct name method to the Person class that should simply return the name attribute.